src/TrafficManagerManagement2/Microsoft.Azure.Management.TrafficManager.nuspec

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata minClientVersion="2.5">
+    <id>Microsoft.Azure.Management.TrafficManager</id>
+    <title>Microsoft Azure WATMv2 Management Library</title>
+    <releaseNotes>WATMv2 Service / CSM-compatible preview library</releaseNotes>
+    <version>$version$</version>
+    <authors>Microsoft</authors>
+    <owners>azure-sdk, Microsoft</owners>
+    <licenseUrl>http://aka.ms/windowsazureapache2</licenseUrl>
+    <projectUrl>https://github.com/WindowsAzure/azure-sdk-for-net</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288890</iconUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <summary>DO NOT SHIP IN PUBLIC. MICROSOFT CONFIDENTIAL. Provides management capabilities for WATMv2 services.</summary>
+    <description>DO NOT SHIP IN PUBLIC. MICROSOFT CONFIDENTIAL. 
+CSM/Resource Groups-compatible library.
+Provides management capabilities for WATMv2 services.</description>
+    <copyright>Copyright © Microsoft Corporation</copyright>
+    <tags>Microsoft Azure WATMv2 management "WATMv2" "WATMv2 management" REST HTTP client windowsazureofficial</tags>
```

Comment:
I changed our code word for the service name. Let me know if that is OK. Should I refer to WATM? What is the difference between doing it in the title or tags?